controls/pcidss_4.yml

```diff
@@ -2174,34 +2174,43 @@ controls:
       status: manual
 
   - id: 8.4
-    title: 'Multi-factor authentication (MFA) is implemented to secure access into the CDE.'
+    title: Multi-factor authentication (MFA) is implemented to secure access into the CDE.
     levels:
       - base
-    status: automated
-    rules:
-      - var_smartcard_drivers=cac
-      - install_smartcard_packages
-      - configure_opensc_card_drivers
-      - service_pcscd_enabled
-      - sssd_enable_smartcards
-      - force_opensc_card_drivers
+    status: supported
+    notes: |-
+      This parent requirement does not set one specific combination of Multi-factor authentication
+      (MFA), so we can't enforce the use of smartcards or any specific solution. The systems
+      usually support MFA but the chosen solution depends on site policies.
     controls:
+    - id: 8.4.1
+      title: MFA is implemented for all non-console access into the CDE for personnel with
+        administrative access.
+      levels:
+        - base
+      status: supported
+      related_rules:
+        - var_smartcard_drivers=cac
```

Comment:
We should double check DoD specificity of `cac` is acceptable for this.

Reply:
This rule is in `related_rules` so it won't have any effect in profiles using this control file. It is there for reference only.